tools/WinObjC.Packaging/Package.Default.props

```diff
@@ -18,7 +18,6 @@
     <_ImportLibsFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\*.lib"/>    
     <_ImportLibsFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\$(Configuration)\*.lib"/>
 
-    <_XamlFilesFromPackage Include="$(MSBuildThisFileDirectory)\xaml\**\*.xaml" />
```

Comment:
the changes we made to WinObjC.Frameworks.targets?

---
In reply to: [108993288](https://github.com/Microsoft/WinObjC/pull/2372#discussion_r108993288) [](ancestors = 108993288)